apps/web/app/hooks/features/useActiveTeam.ts

```
@@ -0,0 +1,44 @@
+	const { timerStatus, stopTimer } = useTimer();
+	const t = useTranslations();
+	const { toast } = useToast();
+	const onChangeActiveTeam = useCallback(
+		(item: TeamItem) => {
+			if (item.data) {
+				/**
+				 * If timer started in Teams and user switches the Team, stop the timer
+				 */
+				if (
+					timerStatus &&
+					timerStatus?.running &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)